lib/converters/converter-v2-to-v1.js

```diff
@@ -116,6 +116,11 @@ _.extend(Builders.prototype, {
         return _.map(item.request.body[mode], function (elem) {
             elem = _.clone(elem);
 
+            if (elem && elem.type === 'file' && _.has(elem, 'src')) {
+                elem.value = elem.src;
```

Comment:
No, we should not do that. `elem.value` is only used when runtime adds a file body (a stream or a buffer) to the element. When exporting/importing and serializing data, file types are only supposed to have a `src`.